release_docs/RELEASE.txt

```diff
@@ -149,8 +149,18 @@ Bug Fixes since HDF5-1.13.3 release
 ===================================
     Library
     -------
-    -
- 
+
+    - Fix CVE-2018-17439 / GHSA-vcxv-vp43-rch7 / HDFFV-10589 / Bug #2226.
```

Comment:
HDFFV-10589 / Bug #2226 should be in the Signature at the bottom of the entry according to our convention, separated by ',':
HDFFV-10589, GH-2226